Bottom: f6a2e4123840c8381cf07c5afb32e9240d4493c2
Top:    34b811fb3d8d447d0e98df307f20ad015a3fa758
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-07 01:47:59 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 795b9fe..c4b314e 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
